### PR TITLE
Adds news template, adds section to dom helpers, adds article-list.js…

### PR DESCRIPTION
**Pull request by tmorris-adobe**
Adds news list template with associated js and css, also adds article-list.js for news page handling/pagination.

Still requires further CSS work, pushing to take advantage of Daves CSS updates

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--octoral--aemsites.hlx.live/en/news/
- After: https://issue21-news--octoral--aemsites.hlx.live/en/news/

### CODE REVIEW

**Comment by tmorris-adobe** [comment]
Test URLs:

Before: https://main--octoral--aemsites.hlx.live/news
After: https://issue21-news--octoral--aemsites.hlx.live/news

File: templates/news-list/news-list.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        width: auto;
+       
+        
+    }
+
+    .card {
+        width: auto;
+    }
+
+    .card h3 {
```
Comment (dave-fink) [inline templates/news-list/news-list.css]: awesome nesting! note you can nest the h3 inside of the .card selector, that's the power of nesting, you only write the selector once, then all the nested elements go inside.  Helps keep things clean and segmented :)

File: templates/news-list/news-list.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import { loadFragment } from '../../blocks/fragment/fragment.js';
+
+export default async function decorate(doc) {
+  const $page = doc.querySelector('main .section');
+  const $articles = div({ class: 'articles' });
+  const $pagination = div({ class: 'pagination' });
+  const articlesPerPage = Number(getMetadata('articles-per-page'));
+  const paginationMaxBtns = Number(getMetadata('pagination-max-buttons'));
+  // Old nav template
+  // const lefNavFrag = await loadFragment('/drafts/Meet/en/products/leftdiv');
```
Comment (dave-fink) [inline templates/news-list/news-list.js]: Lets clean up unused code - keep comments useful :)

File: templates/news-list/news-list.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  // Old nav instance
+  // const $leftNav = lefNavFrag.querySelector('.accordion-wrapper').cloneNode(true);
+  const $leftNav = lefNavFrag.querySelector('.aside-nav-wrapper').cloneNode(true);
+
+  const $aside = aside(
+    $leftNav,
+  );
+
+  const $articleCard = (article) => div(
+    { class: 'card' },
```
Comment (dave-fink) [inline templates/news-list/news-list.js]: I would keep the attributes on the same line like so 
`div({ class: 'card'},`
 `   a({ href: article.path },`
      
it's a little easier to read.

Comment (tmorris-adobe) [reply templates/news-list/news-list.js]: Okay, I 'll do that. The lint scan got mad at me for that line, not sure why

Comment (dave-fink) [reply templates/news-list/news-list.js]: You can ignore eslint errors when they aren't helpful - just hover and select ignore this error for this file - it'll throw a comment at the top to tell eslint to be quiet

**Comment by tmorris-adobe** [comment]
Updates complete

**Review by dave-fink — APPROVED** [review]

**Pull request merged**